### PR TITLE
Add ZKFair

### PR DESCRIPTION
**Pull request by MortyYi**
This PR adds a upcoming L2 ZKFair (previous StableNet):
ZKFair is the first community ZK-L2 based on Polygon CDK and Celestia DA, championing fairness.
websites: https://zkfair.io/
socialMedia: https://docs.zkfair.io/
documentation: https://twitter.com/ZK_fair/
testnet is up, mainnet up due to Dec 20.

### CODE REVIEW

File: packages/config/src/layer2s/zkfair.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+export const zkfair: Layer2 = {
+  isUpcoming: true,
+  type: 'layer2',
+  id: ProjectId('zkfair'),
+  display: {
+    name: 'ZKFair',
+    slug: 'zkfair',
+    description:
+      'ZKFair is the first community ZK-L2 based on Polygon CDK and Celestia DA, championing fairness.',
```
Comment (lucadonnoh) [inline packages/config/src/layer2s/zkfair.ts:14]:
```suggestion
      'ZKFair is the first community Validium based on Polygon CDK and Celestia DA, championing fairness.',
```

**Review by lucadonnoh — CHANGES_REQUESTED** [review]

File: packages/config/src/layer2s/zkfair.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    purpose: 'Universal',
+    category: 'ZK Rollup',
```
[the remark this answers is not in the document]
Comment (MortyYi) [reply packages/config/src/layer2s/zkfair.ts]: Hi, @lucadonnoh. I have fixed this, please check this out.

**Comment by MortyYi** [comment]
Hi, I have merged the master branch.

**Comment by MortyYi** [comment]
Hi, @lucadonnoh. Could you tell me what wrong with the auto merge? It says error happened when run yarn test. But it passes in my local enviroment.

**Comment by lucadonnoh** [comment]
there is no need to merge master everytime, it will be done automatically when put in the merge queue, btw i'll look at the issue

File: packages/config/src/layer2s/zkfair.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    category: 'Validium',
+    dataAvailabilityMode: 'NotApplicable',
+    provider: 'Polygon',
+    links: {
+      websites: ['https://zkfair.io/'],
+      apps: [],
+      documentation: ['https://docs.zkfair.io/'],
+      explorers: [],
+      repositories: [],
+      socialMedia: ['https://twitter.com/ZK_fair/'],
```
Comment (lucadonnoh) [inline packages/config/src/layer2s/zkfair.ts:25]:
```suggestion
      socialMedia: ['https://twitter.com/ZK_fair'],
```

**Review by lucadonnoh — CHANGES_REQUESTED** [review]

File: packages/config/src/layer2s/zkfair.ts (hunk 3 above)
Comment (MortyYi) [reply packages/config/src/layer2s/zkfair.ts]: I have addressed it. thank you very much.

**Comment by MortyYi** [comment]
Hi, @lucadonnoh. Would u mind to make a approval? It is quite urgent for us. Thank you very much.

**Review by lucadonnoh — APPROVED** [review]

**Pull request merged**